### ISSUE TITLE
Change ownership permanently. 

### ISSUE THREAD

**Issue by umop3plsdn** [evilhero/mylar#2125]
so I chown'd both /usr/lib/mylar and /var/lib/mylar and also altered my mylar.service file to my home user and it all works fine... that is until I restart the computer then everything goes back to mylar being the owner and fails to launch the service. Right now I just have a script that changes it all and I manually run it. Is there a way to permanently change the owner for good though?

**Comment by evilhero** [comment]
That sounds like a 3rd party plugin you're running for Mylar that's resetting all the ownership back on either running the service or starting up the system. Mylar doesn't change anything on startup as far as ownership goes, especially the non-user configurable options (it checks to make sure that the logs directory is writable, etc). If you're running something like the synopkg and/or a docker - you'd have to check those respective settings to see what's being enforced (I haven't a clue about either)

**Comment by umop3plsdn** [comment]
This was a straight up install from the AUR in Arch I have no plugins configured whatsoever. It just keeps reverting to the default ownership of mylar:mylar even though I changed them physically in /usr/lib and /var/lib and the service file. Is there any other place it could copy itself from to those directories maybe? I know it uses mono so maybe it's dumping it into those directories every time or something?

**Comment by umop3plsdn** [comment]
Also, this is the behavior of radarr as well

**Comment by evilhero** [comment]
Umm mylar doesn't use mono, and is written in Python. No relation to radarr. And no knowledge of an AUR package - mylar is released via git only. Mylar doesn't copy itself anywhere. It runs where you install it and it stays resident there.

Maybe you should uninstall the pkg and do a git clone of mylar so you can ensure the permissions aren't being modified by something else.

**Comment by umop3plsdn** [comment]
You are correct about the Mono thing I don't know why I just assumed that lol. But still doesn't change why it the ownership defaults to mylar. I just tested it once again the ownership to da5id stuck then I rebooted and checked again and bam mylar in both /var and /usr. This is the AUR package I installed https://aur.archlinux.org/packages/mylar-git/ But i'm going to remove it and just clone the git repo and manually install it like you said. Thanks for your time.
